src/pages/IndividualArticle.jsx

```diff
@@ -74,6 +99,12 @@ export default function IndividualArticle () {
         </div>
         <section className="indent-4 py-4 px-2">{articleData.body}</section>
         <section className="indent-4 p-2">Comments: </section>
+        <form onSubmit={handleSubmit}>
+            <p><label htmlFor="commentBox">Post a comment as: {username}</label></p>
```

Comment:
i don't know if i like you wrapping the label in a p tag. If this is a styling choice then use CSS to style rather than HTML tags.